src/content/docs/service-level-management/faqs-slm.mdx

```diff
@@ -42,10 +42,86 @@ SLI queries support NRDB events. Support for dimensional metrics on SLI queries
 
 ## Can I get alerts on SLI data? [#sli-alerts]
 
-Alerts on New Relic's Service Levels are on our roadmap. 
+Alerts on New Relic's Service Levels are on our roadmap.
 
-## How does New Relic calculate the remaining error budget? [#error-budget] 
+## How does New Relic calculate the remaining error budget? [#error-budget]
 
 The remaining error budget indicates what percentage of requests could still have a bad response over the SLO period without compromising the objective. Therefore, the total amount of tolerated bad responses will vary with the throughput of requests.
 
-Time-based error budgets are on our roadmap.
\ No newline at end of file
+Time-based error budgets are on our roadmap.
+
+## How can I view the queries that define an SLI from the UI?
+
+![SLI edit menu](./images/slm_edit_menu_2.png "SLI edit menu.png")
```

Comment:
I suggest moving the image below the paragraph, just because it's a more usual way to present content.